vmdb/spec/lib/extensions/database_configuration_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "spec_helper"
+
+describe "DatabaseConfiguration patch" do
+  before(:each) do
+    Rails.stub(:env => ActiveSupport::StringInquirer.new("production"))
+
+    @app = Vmdb::Application.new
+    @app.config.paths["config/database"] = "does/not/exist" # ignore real database.yml
+  end
+
+  context "ERB in the template" do
+    before(:each) do
+      @tempfile = Tempfile.new('yml').tap do |f|
+        f.write database_config
+        f.close
+      end
+      @app.config.paths["config/database"].unshift @tempfile.path
+    end
+
+    let(:database_config) { "---\nvalue: <%= 1 %>\n" }
+
+    it "doesn't execute" do
+      expect(@app.config.database_configuration).to eq('value' => '<%= 1 %>')
```

Comment:
I ignored that distinction because it seemed to be covered by the lower level specs of the class. (So I'm just stubbing it as production always.)